datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/dashboard.py

```diff
@@ -0,0 +1,98 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import json
+import os
+import re
+
+import click
+import requests
+
+from ....utils import ensure_dir_exists, path_join, write_file
+from ...constants import get_root
+from ...utils import get_valid_integrations, load_manifest, write_manifest
+from ..console import CONTEXT_SETTINGS, abort
+
+BOARD_ID_PATTERN = r'datadoghq\.com/[^/]+/([^/]+)/.+'
+SCREEN_API = 'https://api.datadoghq.com/api/v1/screen/'
+
+
+@click.group(context_settings=CONTEXT_SETTINGS, short_help='Dashboard utilities')
+def dash():
+    pass
+
+
+@dash.command(context_settings=CONTEXT_SETTINGS, short_help='Export a Dashboard as JSON')
+@click.argument('url')
+@click.argument('integration', required=False)
+@click.pass_context
+def export(ctx, url, integration):
+    if integration and integration not in get_valid_integrations():
+        abort('Unknown integration `{}`'.format(integration))
+
+    org = ctx.obj['org']
+    if not org:
+        abort('No `org` has been set')
+
+    if org not in ctx.obj['orgs']:
+        abort('Selected org {} is not in `orgs`'.format(org))
+
+    org = ctx.obj['orgs'][org]
+
+    api_key = org.get('api_key')
+    if not api_key:
+        abort('No `api_key` has been set')
+
+    app_key = org.get('app_key')
+    if not app_key:
+        abort('No `app_key` has been set')
+
+    match = re.search(BOARD_ID_PATTERN, url)
+    if match:
+        board_id = match.group(1)
+    else:
+        abort('Invalid `url`')
+
+    try:
+        response = requests.get(SCREEN_API + board_id, params={'api_key': api_key, 'application_key': app_key})
+        response.raise_for_status()
+    except Exception as e:
+        abort(str(e).replace(api_key, '*' * len(api_key)).replace(app_key, '*' * len(app_key)))
+
+    payload = response.json()
+    payload['author_info']['author_name'] = 'Datadog'
```

Comment:
For some reason, `author_name` might not exist. So you can probably add `payload.setdefault('author_info', {})`

```
§ ddev meta dash export https://dddev.datadoghq.com/dashboard/4gn-ced-sft/redis---overview-cloned
Traceback (most recent call last):
  File "/Users/alexandre.yang/repos/alexyang/workspaces/integrations-core-ws/.integrations-core.venv/bin/ddev", line 11, in <module>
    load_entry_point('datadog-checks-dev', 'console_scripts', 'ddev')()
  File "/Users/alexandre.yang/repos/alexyang/workspaces/integrations-core-ws/.integrations-core.venv/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Users/alexandre.yang/repos/alexyang/workspaces/integrations-core-ws/.integrations-core.venv/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/alexandre.yang/repos/alexyang/workspaces/integrations-core-ws/.integrations-core.venv/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/alexandre.yang/repos/alexyang/workspaces/integrations-core-ws/.integrations-core.venv/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/alexandre.yang/repos/alexyang/workspaces/integrations-core-ws/.integrations-core.venv/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/alexandre.yang/repos/alexyang/workspaces/integrations-core-ws/.integrations-core.venv/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/alexandre.yang/repos/alexyang/workspaces/integrations-core-ws/.integrations-core.venv/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/alexandre.yang/repos/alexyang/workspaces/integrations-core-ws/.integrations-core.venv/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/alexandre.yang/repos/alexyang/workspaces/integrations-core-ws/integrations-core/datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/dashboard.py", line 63, in export
    payload['author_info']['author_name'] = 'Datadog'
KeyError: 'author_info'
```